src/DynamoCoreWpf/ViewModels/Watch3D/shaderSource/CompileShaders.bat

```diff
@@ -0,0 +1,18 @@
+@echo off
+call "%DXSDK_DIR%Utilities\bin\dx_setenv.cmd"
+pushd %~dp0
+
+REM this is a comment 
+REM https://docs.microsoft.com/en-us/windows/win32/direct3dtools/dx-graphics-tools-fxc-syntax
+REM - /Vi - verbose include info
+REM - /0d - disable optimziations
+REM -/Zi Enable debugging information.
+REM /T fx_5_0 - shader model 5
+REM -/I ./../ include
+REM -  /Fo ..\Resources\_custom.bfx output binary
+REM file to compile Default.fx
+REM - RENAME THESE
+fxc /Vi /Od /Zi /T vs_5_0 /I ./helix_shader_defs/COMMON /Fo ..\compiledShaders\vsDynamoMesh vsDynamoMesh.hlsl
+fxc /Vi /Od /Zi /T ps_5_0 /I ./helix_shader_defs/COMMON /Fo ..\compiledShaders\psDynamoMesh psDynamoMesh.hlsl
```

Comment:
fxc is the directX shader compiler.

Reply:
What's the difference between the .fx shaders and the .hlsl shaders we have now?

Reply:
.fx files are effects files - a higher level directx thing that helix no longer uses.
https://gamedev.stackexchange.com/questions/129827/how-do-hlsl-files-work-together-compared-to-fx

they are kind of like wrappers for pixel and vertex shaders that tie them together.

Reply:
also - 
https://github.com/Microsoft/FX11

effects and the older shader profile are deprecated...